lib/data-loader/dump-data.js

```diff
@@ -20,7 +11,6 @@ require('dotenv').config()
 const baseDir = path.join(__dirname, '..', '..')
 const client = new SiteClient(process.env.DATO_API_TOKEN, { 'X-Reason': 'dump', 'X-SSG': '' })
 const configFile = path.join(baseDir, 'dato.config.js')
-const destinationPath = dumpRoot
 const isPreviewMode = /^(?:staging|development)$/.test(process.env.ENVIRONMENT)
 
-module.exports = () => dump(configFile, client, isPreviewMode, destinationPath)
+module.exports = () => Promise.resolve(configFile, client, isPreviewMode, dumpRoot)
```

Comment:
waarom doe je dit? er wordt nu geen dato content gedumpt, lijkt het?

Reply:
`dumpData()` on line 6 in `bin/dump-data.js` expects it to be a `Promise`

Reply:
that promise is supposed to be returned from `dump`. now nothing is being dumped